main.py

```diff
@@ -1,10 +1,39 @@
-import init_django_orm  # noqa: F401
+import json
+from datetime import datetime
 
+import init_django_orm  # noqa: F401
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        players_data = json.load(file)
+
+    for nickname, values in players_data.items():
+        race = Race.objects.get_or_create(
+            name=values["race"]["name"],
+            description=values["race"]["description"]
+        )[0]
```

Comment:
As an alternative approach to avoid indexing, you could use unpacking
```suggestion
        race, _ = Race.objects.get_or_create(
            name=values["race"]["name"],
            description=values["race"]["description"]
        )
```